modules/proc_configuring-by-using-the-ocp-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-the-notificationsconfiguration-by-using-the-web-console_{context}"]
+= Configuring the NotificationsConfiguration CR by using the web console
+
+You can configure the `NotificationsConfiguration` CR by using the web console.
```

Comment:
I know this section is nested within another one but because it's technically a new section, CR should be spelled out in its first instance. And since it seems to be the only time it's used in this section, it probably doesn't even need an abbreviation:
```suggestion
You can configure the `NotificationsConfiguration` custom resource by using the web console.
```

Reply:
Fixed.